### ISSUE TITLE
ImageNet statistics and data augmentation

### ISSUE THREAD

**Issue by gabridego** [continental/image-statistics-matching#2]
Hello, thanks for the great work!
In the paper it is mentioned that your feature extractor is pretrained on ImageNet, I was wondering if you used ImageNet's mean and standard deviation to normalize the training images after applying your statistics matching, and if you think it would affect the image transformations and the detection performances.
In addition, have you resized the images before forwarding them to the model? Have you used any additional data augmentation technique (e.g. rotation, translation, color jitter, ...)?
Thank you

**Comment by aabramovrepo** [comment]
Hi @gabridego, thanks for your positive feedback! As far as I remember, we used ImageNet's mean and std deviation for normalization (neglecting this would lead to slightly worse results I presume). The training procedure was identical with the one without statistics matching. To your second question: input images were scaled according to standard Faster R-CNN settings, i.e. a shorter side is 600 px (see the paper for more details). We didn't use any additional augmentation techniques, since our goal was to measure the gain of our methods as objective as possible. Applying additional data augmentation would inevitably lead to better results. Hope this helps.

Best,
Alexey

**Comment by gabridego** [comment]
Thanks for the quick and detailed reply, and for your work again! Have a nice day!
